specification/dimensions/adjustmenttype.md

```diff
@@ -0,0 +1,53 @@
+# Adjustment Type
+
+An adjustment is a positive or negative change in cost applied after the original usage or purchase record(s). Adjustments may be related to one or more charges and are identified by the ChargeType column.
+
+Adjustment Type indicates what kind of after-the-fact adjustment the record represents. Adjustment Type is commonly used to identify changes like credits and refunds.
+
+The AdjustmentType column MUST be present and MUST NOT be null or empty when ChargeType is "Adjustment". AdjustmentType MUST be null when ChargeType is not "Adjustment". This column is of type String and SHOULD be one of the example Allowed values or a value of choosing from the vendor.
+
+## Column ID
+
+AdjustmentType
+
+## Display Name
+
+Adjustment Type
+
+## Description
+
+Indicates the record represents an after-the-fact change to an invoice or cost of usage that already occurred (e.g., refunds, credits).
```

Comment:
We usually take the first sentence and remove the column name for descriptions...
```suggestion
A positive or negative change in cost applied after the original usage or purchase record(s).
```